src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -136,6 +137,10 @@ typedef struct uct_ib_mlx5_md {
 } uct_ib_mlx5_md_t;
 
 
+#define UCT_IB_MLX5_MD_FLAGS(_md) ( \
```

Comment:
why needed? who uses this macro without already knowing it's mlx5 device?
is it only for gtest?